readme.md

```diff
@@ -69,6 +69,8 @@ Watch task helps you reduce your efforts during development extensions. If the t
 gulp watch
 ```
 
+In order for Live-reload to work, reload extension manually after starting `gulp watch`.
```

Comment:
As you mentioned, In that case, you need to reload the extension even any websites are already loaded.To be more specific, Please add this `If your extension is already loaded reload extension manually ...` because every extensions are always loaded before as they are in development include me